openapi/paths/tags-configurations.yaml

```
@@ -0,0 +1,54 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Retrieve tags configuration list
+  operationId: GetTagsConfigurationCollection
+  x-sdk-operation-name: getAll
+  description: Retrieve a list of tags configuration.
+  responses:
+    '200':
+      description: List of tags configuration retrieved.
```

nitpick: I think this should be plural
```suggestion
      description: List of tags configurations retrieved.
```